src/PostageStamp.sol

```diff
@@ -430,4 +436,12 @@ contract PostageStamp is AccessControl, Pausable {
         require(ERC20(bzzToken).transferFrom(msg.sender, address(this), amount), "failed transfer");
         pot += amount;
     }
+
+    /**
+     * @notice Set minimum batch depth
+     */
+    function setMinimumBatchDepth(uint8 min) external {
```

Comment:
what about having this an a constructor arg?

Reply:
Done.